portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,30 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private String[] retrieveComments() {
+        ArrayList<String> comments = new ArrayList<String>();
```

Comment:
In Java it is a [good practice](https://stackoverflow.com/questions/147468/why-should-the-interface-for-a-java-class-be-preferred) to use the interface `List` whenever possible.

`List<String> comments = new ArrayList<String>();`

This will make your code more flexible to changing the implementation in the future.

Reply:
Changed!